css/main.css

```diff
@@ -1,3 +1,282 @@
-/* Add your CSS here */
+/* fonts
+font-family: 'Questrial', sans-serif;
+font-family: 'Raleway', sans-serif;
+font-family: 'Roboto', sans-serif;*/
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500;1,400&display=swap');
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+li {
+    list-style: none;
+}
+
+a {
+    text-decoration: none;
+}
+.section {
+    display: flex;
+    margin-right: -2em;
+}
+
+.navbar {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 1rem 1.5rem;
+}
+/* hamburger styles*/
+.hamburger {
+    display: none;
+}
+.bar {
+    display: block;
+    width: 30px;
+    height: 3.5px;
+    margin: 4.5px auto;
+    -webkit-transition: all 0.3s ease-in-out;
```

Comment:
Do you need this browser prefix? What happens if you remove it? Can  https://caniuse.com/ help you decide?